src/runtime_src/core/common/utils.h

```diff
@@ -34,8 +34,11 @@
 #define CU_AP_CONTINUE	(0x1 << 4)
 #define CU_AP_RESET	(0x1 << 5)
 
+std::string convert_version(std::stringstream& version);
```

Comment:
How is this function to be used?  You say it is used from multiple locations, but I have no idea what "current_version" means, what the function returns, or what the argument is.    You can't just add a function in a header file without commenting what it is about.  

Lastly the function cannot be in the global namespace.